int32 request_id
bool result
string error_type